stories/tables/sortable-table.story.js

```diff
@@ -114,8 +114,8 @@ storiesOf('SortableTable', module)
       <Column name="active" component={CustomCellWithRowData} />
     </SortableTable>
   ))
-  .add('with custom row component', () => (
-    <SortableTable data={tableData} rowComponent={CustomRow}>
+  .add('with custom row component and props', () => (
+    <SortableTable data={tableData} rowComponent={CustomRow} rowComponentProps={{onClick: () => { console.log('Clicked') }}}>
```

Comment:
I thought about getting more involved with what this handler did, but I didn't want to invest too much time, as I didn't know if the high-level proposal would be ❤️ 'd. Can update if reviewers feel it's necessary. Lemmeknow!

Reply:
As far as the story goes, I'd like the event handler to reference dynamic data. As implemented, this could be done exclusively through a custom `rowComponent`.

Let's hold off _briefly_ on making this change while we connect on https://github.com/LaunchPadLab/lp-components/pull/589#issuecomment-1559491654.